#
#  Description:  cycle rates for player drivers (in hertz)
#
#  Copyright (C) 2009 Austin Robot Technology                    
#
#  License: Modified BSD Software License Agreement
#
#  $Id: hertz.h 16 2010-01-24 04:32:54Z jack.oquin $
#


#@file
#   
#@brief Cycle rates for various ART vehicle nodes (in hertz).
#

int32 HERTZ_APPLANIX = 20
int32 HERTZ_BRAKE = 20
int32 HERTZ_OBSERVERS = 10
int32 HERTZ_IOADR = 10
int32 HERTZ_LASER_FUSION = 10
int32 HERTZ_LASER_NOISE = 10
int32 HERTZ_MAPLANES = 5
int32 HERTZ_NAVIGATOR = 20
int32 HERTZ_PILOT = 20
int32 HERTZ_PSEUDO_LASER = 10
int32 HERTZ_SICKFAST = 10
int32 HERTZ_STEERING = 20
int32 HERTZ_THROTTLE = 20
float32 HERTZ_VEHICLE_TF = 0.1
int32 HERTZ_VELODYNE_PROC = 10
